Dockerfile

```diff
@@ -4,17 +4,18 @@
 #----------------------------------------------------------------------------------------------
 FROM mcr.microsoft.com/playwright:v1.46.1-jammy
 
-# Install rush
-RUN npm install -g @microsoft/rush
+# Install pnpm
+RUN --mount=type=cache,target=/root/.npm \
+    npm install -g pnpm@10
```

Comment:
Is this necessary? Shouldn't node setup this through corepack now that package.json contains packageManager field?

Reply:
corepack complains about integrity key mismatches, I think playwright needs to be updated to resolve that (to use newer node version). Let's do that in a separate PR since it's a big change.